purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -14,6 +14,9 @@ open class PurchasesConfiguration(builder: Builder) {
     val apiKey: String
     val appUserID: String?
 
+    internal val trimmedApiKey: String
+        get() = apiKey.trim()
```

Comment:
can we just trim the apiKey when setting it in the Builder? That way we don't need to store another API key and we can guarantee `apiKey` will always be trimmed

Reply:
Yeah, I thought about doing this to avoid modifying the input in the `PurchasesConfiguration`.... But I think it should be ok to do that and simplifies things. Will do that, thanks!

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1768/commits/6db9240af99f1bd0ea4fa083ee911892d70ea2b5